src/python/WMCore/WMSpec/Steps/Executors/CMSSW.py

```diff
@@ -371,7 +371,14 @@ def post(self, emulator=None):
     EXIT_STATUS=$?
     if [ $EXIT_STATUS -ne 0 ]; then
        echo "***\nCouldn't untar sandbox with python2: $EXIT_STATUS\n";
-       exit 74;
+       echo "***\nWill try with python2.6 as it is might be old CMSSW release!"
+       python2.6 -m WMCore.WMRuntime.Scripts.UnpackUserTarball $USER_TARBALL $USER_FILES
+       EXIT_STATUS=$?
+       if [ $EXIT_STATUS -ne 0 ]; then
+          echo "***\nCouldn't untar sandbox with python2: $EXIT_STATUS\n";
+          echo "***\nThat was last try... failing..."
```

Comment:
In L374: "it is might be old CMSSW" --> "it might be an old CMSSW" ...

In L378: "sandbox with python2" --> "sandbox with python2.6"

In L379: I think we can remove this "That was last try" line